test/fixtures/common-tests.js

```diff
@@ -1,171 +1,211 @@
 /*global fixture test document*/
 import ReactSelector from '../../';
+import { loadApp } from '../helpers/service-util';
 import { ClientFunction } from 'testcafe';
 
-fixture('ReactJS TestCafe plugin').page('http://localhost:1355');
+const SUPPORTED_VERSIONS = [15, 16];
+
+/*eslint-disable no-loop-func*/
+for (const version of SUPPORTED_VERSIONS) {
+    fixture `ReactJS TestCafe plugin`
```

Comment:
Let's write the React version in the `fixture` name and remove it from the tests names.